graphql/data/schema.js

```diff
@@ -459,7 +460,7 @@ const userType = new GraphQLObjectType({
     backgroundImage: {
       type: backgroundImageType,
       description: "Users's background image",
-      resolve: (user, args, context) => getBackgroundImage(context.user, user),
+      resolve: (user, _args, context) => getBackgroundImage(context.user, user),
```

Comment:
fyi if you didn't know that latest eslint rules want _ prefix if variable isn't used.  I know this is old code but just learned that recently -- similar to python